### PR TITLE
created code of conduct

### PR DESCRIPTION
**Pull request by Davidthecode**
**Purpose of PR?**:

Fixes #

**Does this PR introduce a breaking change?**

**If the changes in this PR are manually verified, list down the scenarios covered:**:

**Additional information for reviewer?** :
_Mention if this PR is part of any design or a continuation of previous PRs_

**Checklist:**
- [ ] Bug fix. Fixes #<issue number>
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update
- [ ] PR Title follows the convention of  `<type>(<scope>): <subject>`

### CODE REVIEW

File: CODE_OF_CONDUCT.md [hunk 1]
```diff
@@ -0,0 +1,119 @@
...
+an individual is officially representing the community in public spaces.
+Examples of representing our community include using an official e-mail address,
+posting via an official social media account, or acting as an appointed
+representative at an online or offline event.
+
+## Enforcement
+
+Instances of abusive, harassing, or otherwise unacceptable behavior may be
+reported to the community leaders responsible for enforcement at
+@sratslla.
```
Comment (rootxrishabh) [inline CODE_OF_CONDUCT.md:63]: Can you please add my name here too?

**Review by rootxrishabh — COMMENTED** [review]
Please check DCO test, it's failing.

**Comment by sratslla** [comment]
Refer this - https://cert-manager.io/docs/contributing/sign-off/

**Comment by Davidthecode** [comment]
i have fixed the issue @sratslla  and @rootxrishabh

**Comment by rootxrishabh** [comment]
LGTM Thanks!

**Pull request merged**